sel4-ftrace-merge/ftrace-merge.c

```diff
@@ -0,0 +1,857 @@
+// SPDX-License-Identifier: GPL-2.0-only
```

Comment:
Perhaps this entire file could benefit if checked with Linux's checkpatch.pl or some sort of autoindentation software?